src/pages/en/comparing-astro-vs-other-tools.md

```diff
@@ -3,32 +3,28 @@ layout: ~/layouts/MainLayout.astro
 title: Astro vs. X
 description: Comparing Astro with other static site generators like Gatsby, Next.js, Nuxt, Hugo, Eleventy, and more.
 ---
+See how Astro compares to several popular site builders and Astro alternatives. Two key features make Astro different from most alternatives:
```

Comment:
Can we keep the original intro, and maybe just lightly rephrase as:

```diff
- We often get asked the question, "How does Astro compare to my favorite site builder, **\_\_\_\_**?" This guide was written to help answer that question for several popular site builders and Astro alternatives.
+ We often get asked the question, "How does Astro compare to my favorite project, **\_\_\_\_**?" This guide was written to help answer that question for several popular site builders and Astro alternatives.
```

The reason is that this page was originally designed for you to jump right to the section that you care about, vs. reading from top to bottom. A compromise might be to link to some "Why Astro" page that we have separately, but I'm not sure if we have something today that fits that bill.

If you really want to call out a few highlights here, then I'd converted them into 1-liners, like:

```
- [Partial hydration](#link)
- [Use your favorite framework](#link)
- [performance optimized for content](#link)
```

Reply:
Done, and agreed on the "Why Astro?" very basic link. This is what I'm taking notes on/for in our shared word document, in a section towards the end, because I do think it will be valuable!

Like with most things, it's easier to write the introduction once you've already finished the essay (because then you know what you're introducing!) And I think once the biggest chunks of these docs are done, it will be more obvious how we've chosen to word and organize these concepts, and a nice intro should more easily fall into place.